roles/cluster_control/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+# tasks file for cluster_control
+
+- name: Try to start master servers
```

Comment:
right now we setup cronjob for our ybmaster and yb-tserver processes. so it keeps running. we need to do systemd route to start/stop process or atleast cronjob.

Reply:
Yes, that is what I also came up with. By researching more on yugabyte DB, I came to notice that a Systemd service has not probably been released yet. I would suggest to create a Systemd service since it might provide a more intelligent way to restart the service on failure.

Reply:
I developed a cronjob script for master and tservers, which is being installed on to the end servers via Ansible.
`cluster_control/files/restart.sh & cluster_control/tasks/crontab_configure.yml`